docs/source/conf.py

```diff
@@ -26,7 +26,7 @@
 sys.path.insert(0, os.path.abspath(PATH_ROOT))
 
 FOLDER_GENERATED = 'generated'
-SPHINX_MOCK_REQUIREMENTS = int(os.environ.get('SPHINX_MOCK_REQUIREMENTS', True))
+SPHINX_MOCK_REQUIREMENTS = int(os.environ.get('SPHINX_MOCK_REQUIREMENTS', 1))
```

Comment:
why is it even an int in the first place lol 
it's only used as a bool :)

Reply:
env variable is string only you canot define bool :P

Reply:
ah right